web/src/poll_modal.ts

```diff
@@ -21,33 +21,33 @@ function add_option_row(e) {
     create_option_row($(e.target));
 }
 
-function delete_option_row(e) {
+function delete_option_row(e: {target: JQuery<HTMLInputElement>}): void {
     const $row = $(e.target).closest(".option-row");
     $row.remove();
 }
 
-export function poll_options_setup() {
+export function poll_options_setup(): void {
     const $poll_options_list = $("#add-poll-form .poll-options-list");
     const $submit_button = $("#add-poll-modal .dialog_submit_button");
     const $question_input = $("#add-poll-form #poll-question-input");
 
     // Disable the submit button if the question is empty.
     $submit_button.prop("disabled", true);
     $question_input.on("input", () => {
-        if ($question_input.val().trim() !== "") {
+        const inputValue = $question_input.val();
+        if (inputValue !== null && typeof inputValue === "string" && inputValue.trim() !== "") {
             $submit_button.prop("disabled", false);
         } else {
             $submit_button.prop("disabled", true);
         }
     });
-
     $poll_options_list.on("input", "input.poll-option-input", add_option_row);
     $poll_options_list.on("click", "button.delete-option", delete_option_row);
 
     // setTimeout is needed to here to give time for simplebar to initialise
     setTimeout(() => {
-        Sortable.create($("#add-poll-form .poll-options-list .simplebar-content")[0], {
-            onUpdate() {},
+        SortableJS.create($("#add-poll-form .poll-options-list .simplebar-content")[0], {
+            // onUpdate() {},
```

Comment:
What's the story with commenting this line?

Reply:
The onUpdate() {} function threw an "Empty function Exception" when parsed through lint. Therefore, I had to comment it out.

Reply:
Read the linter error more carefully. https://eslint.org/docs/latest/rules/no-empty-function makes clear that adding a comment is a better fix.

Reply:
Hi Tim,
Thanks for pointing it out. I made the required changes.